src/server.ts

```diff
@@ -27,7 +28,7 @@ const regionConfig = regionConfiguration(region);
 
 const defaultOptions: https.RequestOptions = {
   auth: `${merchantId}:${secretKey}`,
-  hostname: 'global-api-sandbox.afterpay.com'
+  hostname: hostname
```

Comment:
Even this would work I think.

```suggestion
  hostname
```